content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -88,6 +88,147 @@ Once the installation is finished, RDI is ready for use.
 
 {{<note>}}RDI gives you instructions to help you create secrets and create your pipeline.{{</note>}}
 
+## "Silent" installation
+
+You can use the
+[`redis-di install`]({{< relref "/integrate/redis-data-integration/ingest/reference/cli/redis-di-install" >}})
+command with the `--file` option (or the shorter version `-f`) to supply answers
+to the installer's questions automatically from a [TOML](https://toml.io/en/) file:
+
+```bash
+redis-di install --file silent.toml
+```
+
+The following example shows the properties you can use:
+
+```toml
+title = "RDI Silent Installer Config"
+
+high_availability = false
+scaffold = false
+deploy = true
+db_index = 4
+deploy_directory = "/opt/rdi/config"
+
+[rdi.cluster]
+host = "localhost"
+port = 9443
+username = "username"
+password = "password"
+
+[rdi.database]
+host = "localhost"
+port = 12001
+username = "username"
+password = "password"
+use_existing_rdi = true
+ssl = true
+
+[rdi.database.certificates]
+ca = "/home/ubuntu/rdi/certs/ca.crt"
+cert = "/home/ubuntu/rdi/certs/client.crt"
+key = "/home/ubuntu/rdi/certs/client.key"
+passphrase = "foobar"
+
+# WARNING! We provide the properties below for special
+# use cases. However, we *strongly* recommend that you don't
+# store the source and target secrets in the clear like
+# this for normal purposes!
+
+# [sources.default]
+# username = "username"
+# password = "password"
+# ssl = false
+
+# [sources.default.certificates]
+# ca = "/home/ubuntu/rdi/certs/ca.crt"
+# cert = "/home/ubuntu/rdi/certs/client.crt"
+# key = "/home/ubuntu/rdi/certs/client.key"
+# passphrase = "foobar"
+
+# [targets.default]
+# username = "username"
+# password = "password"
+# ssl = false
+
+# [targets.default.certificates]
+# ca = "/home/ubuntu/rdi/certs/ca.crt"
+# cert = "/home/ubuntu/rdi/certs/client.crt"
+# key = "/home/ubuntu/rdi/certs/client.key"
+# passphrase = "foobar"
+```
+
+The tables below describe the properties for each section in more detail:
```

Comment:
the reference table should be in a different snippet and section than the toml file